README.md

```diff
@@ -71,6 +71,36 @@ class App extends React.Component {
 }
 ```
 
+#### With React Hooks
+```
+import React, { useState } from "react";
+import Select from "react-select";
+
+const options = [
+  { value: "chocolate", label: "Chocolate" },
+  { value: "strawberry", label: "Strawberry" },
+  { value: "vanilla", label: "Vanilla" }
+];
+
+export default function App() {
+
+  const [selectedOption, setselectedOption] = useState(null);
+  const handleChange = selectedOption => {
+    setselectedOption(selectedOption);
+  };
+  
+  return (
+    <div className="App">
+      <Select
+        defaultValue={selectedOption}
+        onChange={handleChange}
```

Comment:
```suggestion
        onChange={setSelectedOption}
```
The handle change method seems redundant, unless you add a console log like the class component example